jms/1.7/modules/ROOT/pages/jms-studio-configuration.adoc

```diff
@@ -0,0 +1,478 @@
+[start=5]
+. In the *Producer* tab, set the *Time to live* field to `60000` to define the default time the message is in the broker before it expires and is discarded.
+
+.JMS Connector Producer Global Configuration
+image::jms-global-configuration-producer.png[In the Producer tab, set the Time to live field to define the default time the message is in the broker.]
+
+In the XML editor, the `<jms:config>`,`selector`, and `timeToLive` parameters' configuration looks like this:
+
+[source,xml,linenums]
+----
+<jms:config name="JMS_Config">
+  <jms:active-mq-connection/>
+  <jms:consumer-config selector="JMSType = `INVOICE`"/>
+  <jms:producer-config timeToLive="60000"/>
+</jms:config>
+----
+
+These properties can be overridden at the operation level when required. In the following example, you need to publish a message with a shorter TTL. To override the value of the *Time to live* field set on the global element, follow these steps:
+
+. In Studio, select the *Publish* operation from your flow.
+. In the *Publish* configuration screen, scroll down to the *Publish Configuration* section.
+. Set the *Time to live* field to `10000` to override the default global configuration previously set.
+
+.JMS Connector Publish Configuration Override
+image::jms-publish-configuration-1.png[In the Publish Configuration section, set the Time to Live field to override the default global configuration previously set.]
+
+In the XML editor, the `<jms:publish>` and `timeToLive` parameters configuration looks like this:
+
+[source,xml,linenums]
+----
+<jms:publish config-ref="JMS_Config" timeToLive="10000"/>
+----
+
+[[configure-connection]]
+== Configure the Connection to a Broker
+
+To configure the connection in the global element, define the connection by selecting either *ActiveMQ Connection* or *Generic Connection*, set up the connection libraries and the JMS specification. +
+JMS Connector also enables you to provide credentials if you require an authenticated connection, and also to configure connections caching to increase the performance of the application.
+
+=== Configure an Active MQ Connection
+
+JMS Connector supports ActiveMQ 5 by using the *ActiveMQ Connection*. With this connection, you can use `JMS_1_1 (Default)` or `JMS_1_0_2b` specifications, and configure all the general connection parameters for JMS, as well as the custom parameters only present in ActiveMQ.
+
+Once you declare the ActiveMQ connection, set up the *Connection Factory* with your custom configuration. Every parameter in the connection comes with a default value, meaning that you are required to configure only the parameters relevant for your use case. Also, the ActiveMQ connection exposes parameters that are exclusive to the ActiveMQ implementation, like *Initial redelivery delay*.
+
+In the following example, you configure an *ActiveMQ Connection*:
+
+. In Studio, navigate to the *Global Elements* tab.
+. Click *Create*.
+. In the filter box type `jms`, and select *JMS Config*.
```

Comment:
Are both actions in the filter box? Maybe this should be two steps if not, just to clarify; otherwise, remove the period.

Reply:
Comma after "filter box"